### PR TITLE
Bump repo config to v0.6.2

### PR DESCRIPTION
**Pull request by TalweSingh**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "submodules/api-common-protos"]
 	path = submodules/api-common-protos
 	url = https://github.com/googleapis/api-common-protos.git
+[submodule "submodules/frequenz-api-common"]
+	path = submodules/frequenz-api-common
+	url = https://github.com/frequenz-floss/frequenz-api-common.git
```
Comment (llucax) [inline .gitmodules]: Recursive dependency? :laughing: :repeat: 

```suggestion
```

File: mkdocs.yml [hunk 2]
```diff
@@ -105,10 +106,10 @@ plugins:
           import:
             # See https://mkdocstrings.github.io/python/usage/#import for details
             - https://docs.python.org/3/objects.inv
+            - https://frequenz-floss.github.io/frequenz-api-common/v0.3/objects.inv
```
Comment (llucax) [inline mkdocs.yml]: :repeat: 
```suggestion
```

**Review by llucax — COMMENTED** [review]
Just remove the self-references and we are ready to go!

File: .gitmodules (hunk 1 above)
Comment (TalweSingh) [reply .gitmodules]: oops  🔁 💥

**Review by llucax — DISMISSED** [review]

**Comment by TalweSingh** [comment]
Can you reapprove this pls? The commit wasn't properly signed. Also can you update the branch proteciton rules here as well, don't have access.

**Review by tiyash-basu-frequenz — DISMISSED** [review]

**Comment by tiyash-basu-frequenz** [comment]
Looks like merging a few dependabot PRs are causing a conflict here. Needs a rebase.

**Review by tiyash-basu-frequenz — DISMISSED** [review]

**Review by tiyash-basu-frequenz — APPROVED** [review]

**Pull request merged**